auth/tenant_mgt.go

```diff
@@ -240,6 +242,11 @@ func (t *TenantToCreate) EnableEmailLinkSignIn(enable bool) *TenantToCreate {
 	return t.set(enableEmailLinkSignInKey, enable)
 }
 
+// EnableAnonymousUsers enables or disables anonymous user.
```

Comment:
@egilmorez Hi Eric, please take a look at the doc strings when you get a chance. Thank you!

Reply:
You bet, will add Kevin as well.

Reply:
Thank you!